src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/JsonTest/JsonTest.cs

```diff
@@ -286,5 +296,40 @@ public async Task TestJsonReadAsync()
                 }
             }
         }
+
+        [ConditionalFact(typeof(DataTestUtility), nameof(DataTestUtility.IsJsonSupported))]
+        public void TestNullJson()
+        {
+            string tableName = DataTestUtility.GetUniqueNameForSqlServer("Json_Test");
+
+            string tableCreate = "CREATE TABLE " + tableName + " (Data json)";
+            string tableInsert = "INSERT INTO " + tableName + " VALUES (@jsonData)";
+            string tableRead = "SELECT * FROM " + tableName;
+
+            using (SqlConnection connection = new SqlConnection(DataTestUtility.TCPConnectionString))
```

Comment:
Consider using modern C# constructs:

```c#
using SqlConnection connection = new SqlConnection(DataTestUtility.TCPConnectionString);
```

This reduces the unneeded nesting etc.

Reply:
modified